node/src/chain_spec/dev.rs

```diff
@@ -54,6 +54,7 @@ pub fn dev_config(
                 .into_iter()
                 .map(|acc| EndowedAccountWithBalance(acc, INITIAL_BALANCE))
                 .collect(),
+                #[cfg(not(feature = "without-sudo"))]
```

Comment:
Oh my fault, because of this
<img width="982" alt="Bildschirmfoto 2022-07-08 um 09 18 26" src="https://user-images.githubusercontent.com/33418719/177938675-30d23ec1-c473-4660-b62d-546da0a04ce8.png">

Reply:
This is not the case, the next line after the feature is `get_account_id_from_seed::<sr25519::Public>("Alice"),`, which defines `Alice` as the superuser account.